Gems/PhysX/Code/Tests/RagdollTests.cpp

```diff
@@ -367,4 +367,19 @@ namespace PhysX
         float minZ = ragdoll->GetAabb().GetMin().GetZ();
         EXPECT_NEAR(minZ, 0.0f, 0.05f);
     }
+
+    TEST(ComputeHierarchyDepthsTest, DepthValuesCorrect)
```

Comment:
It'd be nice to have also unit tests covering the clamping of masses for nodes are working as expected.